study.md

```diff
@@ -27,17 +27,24 @@ it's also not a rendered page.
 Please name two things the DOM is not.
 
 ```md
-<!-- your answer here -->
+<!-- In order to explain what the DOM is not, I will phrase what the DOM is. The Dom is
+a Document Object Model(a tree structure) that allows us to connect with web pages and this connection serves
+to let us modify the content of HTML documets.
+The DOM is not a literal explicit HTML file
+It is not a literal CSS file
+It is not what we called as source in a website.
+ -->
 ```
 
 Is the information inside your Dev Tools the DOM? Why or why not?
 
 ```md
-<!-- your answer here -->
+<!-- Yes, when we inspecg a website, we get a visual representation of the DOM that is as to say a visual
```

Comment:
The DOM is an interface that represents HTML documents in a tree structure.  What you see in your web browser's developer tools is a visual representation of the browser's implementation of that interface, but not the DOM itself.